### ISSUE TITLE
Refine Synapse data access language and add popover for individual file access

### ISSUE THREAD

**Issue by adamjtaylor** [ncihtan/htan-portal#541]
Hey @inodb, in chatting with the Sage Governance team lately we would like to ensure that users see some text on needing to be registered and logged into Synapse to access files, and possibly some language on data use.

There is already a download dialogue when multiple files are selected which we can refine.

When individual filenames are clicked this currently goes directly to a link to Synapse. Here we would propose adding a popover as seen for dbGaP access which could then include both the Synapse link and programmatic access options.

Let's setup some time to discuss in an upcoming operations meeting.

Linking back to a Sage internal Jira ticket that this is tracked in for the governance team: [HSD-288 ](https://sagebionetworks.jira.com/browse/HSD-288)

**Comment by inodb** [comment]
@adamjtaylor @aclayton555 
> we would like to ensure that users see some text on needing to be registered and logged into Synapse to access files

It looks like there is a tooltip now in Synapse:
<img width="304" alt="image" src="https://github.com/ncihtan/htan-portal/assets/1334004/0bcde778-5324-414a-b559-f3ac84d4ea82">

Do we still want to add language about needing to register? This seems pretty clear to me


> and possibly some language on data use.

We can maybe add a banner site-wide that explains data usage agreements across all sources rather than having it as a pop-up for each file?

**Comment by aclayton555** [comment]
Based on a conversation that Adam and Ashley had with the Sage Governance team on 2024.01.12: The new tooptip in Synapse is an improvement to the user experience. Adam will be preparing a screen recording of the current external user experience from the HTAN Data Portal to Synapse. This will be taken into consideration in ongoing work by our platform team to improve user navigation, and support awareness among user that they need to perform certain actions before they can access files (e.g. creating and/or logging into their Synapse account). 

As for any sort or data use terms, Sage Governance would like to hold on this for now. Consider revisiting this in the renewal, and any sort of banners we may want to implement.

TL:DR no further work is needed on this at this time.
